package.json

```diff
@@ -15,10 +15,15 @@
     "@mui/icons-material": "^5.15.4",
     "@mui/material": "^5.15.4",
     "axios": "^1.6.4",
+    "localforage": "^1.10.0",
+    "match-sorter": "^6.3.3",
```

Comment:
Why do you need that?

Reply:
![image](https://github.com/tomekwoojcik/my-online-shop/assets/145002697/9301e501-4a00-4913-a420-3ca2c68c3be7)
Probably libraries are required by react-router-dom

Reply:
<img width="679" alt="Screenshot 2024-02-09 at 20 21 50" src="https://github.com/tomekwoojcik/my-online-shop/assets/145002697/f819ed64-3d59-4350-ae4c-ae3495dac210">
Can be deleted. FYI :)